manifest/templates/backstage_env_secret.tmpl

```diff
@@ -1,13 +1,15 @@
-BASE_URL="https://backstage-<NAMESPACE>.<OCP_CLUSTER_DOMAIN>/" where <OCP_CLUSTER_DOMAIN> corresponds to the OpenShift domain: apps.qshift.snowdrop.dev, apps.newqshift.lab.upshift.rdu2.redhat.com, etc
-BACKSTAGE_AUTH_SECRET="Execute this command from a terminal to get a crypto password: node -p 'require("crypto").randomBytes(24).toString("base64")'>"
+APP_BASE_URL="https://backstage-<NAMESPACE>.apps.<OCP_CLUSTER_DOMAIN>/" where <OCP_CLUSTER_DOMAIN> corresponds to the OpenShift domain: apps.qshift.snowdrop.dev OR "http://localhost:3000"
```

Comment:
It should be nice to propose here local address (https://localhost:3000) also in case where we are running backstage locally

Reply:
Good catch. As `envsubst` only replace/substitute values, we cannot really include logic able to deal with: local vs remote, etc as we can have with a bash, python, script, etc. 

We discussed such an idea within this PR:  https://github.com/q-shift/backstage-playground/pull/14 and Ioannis did a proposition. I propose to add this bash script part of another PR and to review the needed changes separately. do you agree ? @aureamunoz

Reply:
ok